pom.xml

```diff
@@ -204,6 +204,13 @@
       <version>${log4j.version}</version>
       <scope>test</scope>
     </dependency>
+    <!-- route slf4j 2 over log4j2 during integration tests -->
```

Comment:
@CodePrometheus fyi I added this to kill the warning in the integration test for pulsar. unrelated I know but too lazy to make a separate PR for such a nit.